src/main/java/frc/robot/subsystems/intake/pivot/PivotState.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems.intake.pivot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public enum PivotState {
+
+	ON_FLOOR(Rotation2d.fromDegrees(0)),
```

Comment:
Does it physically touching the floor?

Reply:
not calibrated yet, but i dont think so

Reply:
If it doesn't so it may be a misleading name. If you disagree then mark as resolved.

Reply:
consider changing the name to `COLLECTING` or something